.github/workflows/validatecrd.yml

```diff
@@ -0,0 +1,19 @@
+name: validate crd
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: nolar/setup-k3d-k3s@v1
+        with:
+          # This is the Kubernetes version used by k3d. See https://kubernetes.io/releases/
+          version: v1.26.5
+      - name: apply crd
+        run: kubectl apply -f config/crd/bases/operator.kyma-project.io_eventings.yaml
```

Comment:
Is it a required job?

Reply:
No, I do not have the permissions to make this job required.